### ISSUE TITLE
nerves_system_br dependency issue

### ISSUE THREAD

**Issue by alecho** [LeToteTeam/kiosk_system_rpi3#26]
I ran into the following error when adding this to my project.
```
Failed to use "nerves_system_br" because
  kiosk_system_rpi3 (version 1.0.0) requires ~> 1.0.0
  mix.lock specifies 1.3.2
```

I saw that the `master` branch had bumped the nerves_system_br dependency so I set it to use `branch: "master"` but then of course it couldn't find the prebuilt archive:

```
** (Mix)
The following Nerves packages need to be built:

  kiosk_system_rpi3
```

Can I get away with unlocking nerves_system_br and using a 1.0.x version? Should I wait for the 1.3 dependency bump to hit a release with an archive. I'm not brave enough to try and build the image in Docker myself.

**Comment by mobileoverlord** [comment]
Ah yeah, thanks for reminding me. I'll start to make a release today. I had to fix the issues with building on circle and that seems to be working now.

**Comment by mobileoverlord** [comment]
The nerves_system_br dependency was fixed in the 1.1.0 release
